brozzler/js-templates/umbraBehavior.js.j2

```diff
@@ -54,25 +55,27 @@ class UmbraBehavior {
                 // console.log("exception looking at iframe" + iframes[i] + ": " + e);
             }
         }
+
         var documentsLength = documents.length;
         for (var j = 0; j < documentsLength; j++) {
             if (closeSelector) {
                 var closeTargets = documents[j].querySelectorAll(closeSelector);
-                if ((closeTargets.length > 0) &&
-                    (this.alreadyDone.indexOf(closeTargets[0]) === -1) &&
-                    (this.isVisible(closeTargets[0]))) {
-                    doTarget(closeTargets[0], 'click');
+                for (var i = 0; i < closeTargets.length; i++) {
+                    this.doTarget(closeTargets[i], "click", "close");
```

Comment:
This `type` parameter with values "open"/"close" that change the behavior of `doTarget()` is not the most elegant design imho. Instead, I would just replace this `doTarget()` call with `closeTargets[i].click()`. Unless we know of close buttons that require a mouseover first? But I doubt it. @galgeek?

Reply:
I like the change you suggest, @nlevitt. 
I don't remember checking whether a mouseover is necessary, with the other behavior I worked on using a closeSelector, and the site is no longer on the live web.